libexec/cli/inspect.exec

```diff
@@ -0,0 +1,85 @@
+# through Lawrence Berkeley National Laboratory (subject to receipt of any
+# required approvals from the U.S. Dept. of Energy).  All rights reserved.
+# 
+# This software is licensed under a customized 3-clause BSD license.  Please
+# consult LICENSE file distributed with the sources of this project regarding
+# your rights to use or distribute this software.
+# 
+# NOTICE.  This Software was developed under funding from the U.S. Department of
+# Energy and the U.S. Government consequently retains certain rights. As such,
+# the U.S. Government has been granted for itself and others acting on its
+# behalf a paid-up, nonexclusive, irrevocable, worldwide license in the Software
+# to reproduce, distribute copies to the public, prepare derivative works, and
+# perform publicly and display publicly, and to permit other to do so. 
+# 
+# 
+
+
+## Basic sanity
+if [ -z "$SINGULARITY_libexecdir" ]; then
+    echo "Could not identify the Singularity libexecdir."
+    exit 1
+fi
+
+## Load functions
+if [ -f "$SINGULARITY_libexecdir/singularity/functions" ]; then
+    . "$SINGULARITY_libexecdir/singularity/functions"
+else
+    echo "Error loading functions: $SINGULARITY_libexecdir/singularity/functions"
+    exit 1
+fi
+
+if [ -f "${HOME:-}/.singularity-init" ]; then
+    . "${HOME:-}/.singularity-init"
+fi
+
+# Default inspect is labels, unless client asks for runscript
+SINGULARITY_INSPECT_SCRIPT="/singularity"
+export SINGULARITY_INSPECT_SCRIPT
+if [ -f "$SINGULARITY_libexecdir/singularity/cli/action_argparser.sh" ]; then
+    . "$SINGULARITY_libexecdir/singularity/cli/action_argparser.sh"
+else
+    message ERROR "Could not find the action argument parser\n"
+    exit 1
+fi
+
+
+if [ -z "${SINGULARITY_IMAGE:-}" ]; then
+    SINGULARITY_IMAGE="${1:-}"
+    export SINGULARITY_IMAGE
+    shift
+fi
+
+
+if [ -z "${SINGULARITY_IMAGE:-}" ]; then
+    if [ -e "$SINGULARITY_libexecdir/singularity/cli/$SINGULARITY_COMMAND.help" ]; then
+        head -n 1 "$SINGULARITY_libexecdir/singularity/cli/$SINGULARITY_COMMAND.help"
+    else
+        message ERROR "To see usage summary, try: singularity help $SINGULARITY_COMMAND\n"
+    fi
+    exit 0
+fi
+
+if [ -f "$SINGULARITY_sysconfdir/singularity/init" ]; then
+    . "$SINGULARITY_sysconfdir/singularity/init"
+fi
+
+
+if [ -x "$SINGULARITY_libexecdir/singularity/image-handler.sh" ]; then
+    . "$SINGULARITY_libexecdir/singularity/image-handler.sh"
+fi
+
+if [ -z "${SINGULARITY_NOSUID:-}" -a -u "$SINGULARITY_libexecdir/singularity/bin/action-suid" ]; then
+    exec "$SINGULARITY_libexecdir/singularity/bin/action-suid" cat "${SINGULARITY_INSPECT_SCRIPT}" <&0
+elif [ -x "$SINGULARITY_libexecdir/singularity/bin/action" ]; then
+    exec "$SINGULARITY_libexecdir/singularity/bin/action" cat  "${SINGULARITY_INSPECT_SCRIPT}" <&0
+else
+    message ERROR "Could not locate the Singularity binary: $SINGULARITY_libexecdir/singularity/bin/action\n"
+    exit 1
+fi
```

Comment:
Should I wait until you are ready to do the pull, and I will make the changes locally? Otherwise, I can implement on my branch, and you will have to merge on your end. either way works for me!